src/test/java/com/selfxdsd/selfweb/api/output/JsonTaskTestCase.java

```diff
@@ -189,28 +158,5 @@ public void hasNullDeadline() {
             Matchers.equalTo("null")
         );
     }
-        
-        /**
-     * JsonTask hasn't estimation
-     */
-    
-        
-        @Test
-        public void hasNullEstimation() {
```

Comment:
@Lumi3011 I removed this test since it's the same as "hasEstimation" (only the value is 0 instead of 60)